src/screens/AdminPortal/PluginStore/components/UninstallConfirmationModal.tsx

```
@@ -1,76 +1,85 @@
-      </DialogActions>
-    </Dialog>
+      <div data-testid="uninstall-modal">
+        <p className="mb-2" data-testid="uninstall-modal-title">
+          {t('uninstallPlugin.message', { name: plugin.name })}
+        </p>
+        <p className="text-secondary small">{t('uninstallPlugin.warning')}</p>
+      </div>
+    </BaseModal>
   );
-}
+}
```

ESLint

[error] Insert `⏎`

(prettier/prettier)